aggregator/src/blob_consistency/avail.rs

```diff
@@ -0,0 +1,95 @@
+use super::{AssignedBlobDataExport};
+use crate::{BatchData, RlcConfig};
+use eth_types::{H256, U256};
+use halo2_base::{gates::range::RangeConfig, AssignedValue, Context};
+use halo2_ecc::bigint::CRTInteger;
+use halo2_proofs::{
+    circuit::{AssignedCell, Layouter, Value},
+    halo2curves::bn256::Fr,
+    plonk::{ConstraintSystem, Error, Expression},
+};
+use serde::{Deserialize, Serialize};
+use snark_verifier_sdk::LIMBS;
+use zkevm_circuits::{table::U8Table, util::Challenges};
+
+pub const BLOB_WIDTH: usize = 4096;
+
+#[derive(Debug, Clone)]
+pub struct BlobConsistencyConfig<const N_SNARKS: usize> {}
+
+impl<const N_SNARKS: usize> BlobConsistencyConfig<N_SNARKS> {
+    pub fn construct(
+        _meta: &mut ConstraintSystem<Fr>,
+        _challenges: &Challenges<Expression<Fr>>,
+        _u8_table: U8Table,
+        _: RangeConfig<Fr>,
+    ) -> Self {
+        unimplemented!()
```

Comment:
When I ran the test command `cargo test --release --workspace --exclude integration-tests --exclude circuit-benchmarks --features da-avail batch_circuit`, three tests failed due to this `construct` method being unimplemented. The three tests are:
`aggregation::config::batch_circuit_degree`
`tests::aggregation::batch_circuit_encode`
`tests::aggregation::batch_circuit_raw`

Is this fn supposed to be left unimplemented or should it actually return some default placeholder value?

Reply:
this is just a placeholder for the da-avail circuits. once those are added, those tests (and all others) will be expected to pass with the command you used.